ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/hypershift/provision/cucushift-installer-rehearse-aws-ipi-ovn-hypershift-provision-chain.yaml

```diff
@@ -0,0 +1,18 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-provision
+  steps:
+  - chain: ipi-conf-aws
+  - ref: ovn-conf
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  - chain: hypershift-install
```

Comment:
You can wrapper the above steps with `- chain: cucushift-installer-rehearse-aws-ipi-ovn-provision`